runtime/tools/python/test/test_run.py

```diff
@@ -47,6 +47,7 @@ def test_flatbuffer_cmd():
     ), f"one of more tests failed in={test_flatbuffer_cmd.__name__}"
 
 
+@pytest.mark.skip
 def test_dir_flatbuffer():
```

Comment:
Why are we disabling test_dir_flatbuffer? Shouldn't we make sure that this test always passes as it runs all the Silicon tests on device? I don't think this test uses any perf features?